.github/workflows/packaging.yml

```diff
@@ -0,0 +1,54 @@
+name: Packaging release assets
+
+on:
+  push:
```

Comment:
I think we will also need to use `on.release` soon, though I need to check exactly how it works. 

When we choose to start a release we should probably branch with `prerelease-X.Y.Z` and then start all the additional tests, check integration with cutter, r2dec, extras, etc. At that point, we should fix possible issues in master and cherry-pick the commits in the `prerelease-X.Y.Z` branch as well (which will run again all the tests and integration with other repositories). Once all integration tests are fixed, we can create `release-X.Y.Z` branch.